cmd/vpu_plugin/vpu_plugin.go

```diff
@@ -179,6 +186,10 @@ func main() {
 	}
 
 	plugin := newDevicePlugin(ctx, vendorID, productIDs, sharedDevNum)
+	if plugin == nil {
+		klog.Fatal("Cannot create device plugin, please check above error messages.")
+		os.Exit(1)
```

Comment:
`klog.Fatal()` exits so `os.Exit(1)` is not needed

Reply:
done.